### PR TITLE
Updates sssd to support a variety of conf parameters

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -18,19 +19,20 @@ install:
   - ps: |
       $null = `
         Start-Process -FilePath "${env:temp}\salt-minion.exe" `
-        -ArgumentList  @("/S", "/install-dir=c:\salt") -NoNewWindow -PassThru -Wait
-  - ps: C:\salt\salt-call.bat --versions-report
+        -ArgumentList "/S" -NoNewWindow -PassThru -Wait
+  - ps: |
+      & "C:\Program Files\Salt Project\Salt\salt-call.bat" --versions-report
```
Comment (lorengordon) [inline appveyor.yml:24]: With salt 3006, it's `.exe`, not `.bat`......

Comment (eemperor) [reply appveyor.yml:24]: 🤦

**Review by lorengordon — COMMENTED** [review]
Approach looks good to me. Would like to see updates to the readme and pillar.example to account for the new config key....

File: README.md [hunk 2]
```diff
@@ -184,6 +184,22 @@ connector-software files (e.g. `/var/lib/pbis`)
...
+
+- *`sssd_conf_parameters`*:
+```yaml
+sssd_conf_parameters   
+  <key1>: '<value1>'
+  ...
+  <keyN>: '<valueN>'
+```
+
+A variety of supported parameters can used to customize the operation of `sssd`.
```
Comment (lorengordon) [inline README.md:201]:
```suggestion
Any parameter supported by `sssd` may be used to customize the operation of `sssd`.
```

Comment (eemperor) [reply README.md]: I used your suggesting with a slight change.

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**